control_plane/src/tenant_migration.rs

```diff
@@ -93,6 +93,22 @@ pub fn migrate_tenant(
     // Get a new generation
     let attachment_service = AttachmentService::from_env(env);
 
+    fn build_location_config(
+        mode: LocationConfigMode,
+        generation: Option<Generation>,
```

Comment:
I wonder if this should be `Option<u32>` instead of `Option<Generation>`. It might be fine to have former, but latter makes also a lot of sense.

Reply:
Use of Generation in the models.rs LocationConf type was inconsistent with use of u32 elsewhere in that file, I've added a commit that cleans it up to use u32 consistently in models.rs (and thereby also here)